src/app/core/store/projects/projects.effects.ts

```diff
@@ -71,15 +75,18 @@ export class ProjectsEffects {
 
   @Effect()
   public update$: Observable<Action> = this.actions$.ofType<ProjectsAction.Update>(ProjectsActionType.UPDATE).pipe(
-    switchMap(action => {
-      const organizationCode = action.payload.project.organizationCode;
+    withLatestFrom(this.store$),
+    switchMap(([action, state]) => {
+      const organization = state.organizations.entities[action.payload.project.organizationId];
+      const project = state.projects.entities[action.payload.project.id];
```

Comment:
I assume this is needed just because we want to use the old project code for the REST call. In that case I would recommend to call it either `oldProject` or `previousProjectCode` in order to be self-explaining when we modify this code in the future.

Reply:
Okay